fixtures-generator/graphql/responses_generator.py

```diff
@@ -30,15 +31,16 @@ def _to_json(self, object, output_filename):
 
 class ShopListsGenerator(ResponsesGenerator):
 
-    def __init__(self, output_dir, fixture_set):
-        self.__SHOPS_FIXTURE = helpers.get_shopozor_fixture(fixture_set)
-        super().__init__(os.path.join(output_dir, fixture_set, 'Consumer'))
+    def __init__(self, fixtures_dir, output_dir, fixtures_set):
+        super().__init__(fixtures_dir, os.path.join(
+            output_dir, fixtures_set, 'Consumer'), fixtures_set)
+        self.__SHOPS_FIXTURE = helpers.get_shopozor_fixture(self._INPUT_DIR)
 
     def _produce_data(self):
         return {
             'data': {
                 'shops': {
-                    'edges': [helpers.shop_node(shop_fixture['pk'], shop_fixture['fields']) for shop_fixture in [shop for shop in self.__SHOPS_FIXTURE if shop['model'] == 'shopozor.shop']]
+                    'edges': [helpers.shop_node(shop) for shop in self.__SHOPS_FIXTURE['shops']]
```

Comment:
With Hasura, we don't have this notion of "edges" and "nodes" in the graphql schema. We don't have to "follow" edges anymore. Is this intentional ?

Reply:
As I said, it's code ported from the old saleor backend; that kind of details will be solved / removed when I write the api tests.